cart/views.py

```diff
@@ -1,3 +1,71 @@
-from django.shortcuts import render
+import json, bcrypt
 
-# Create your views here.
+from django.views     import View
+from django.http      import JsonResponse
+
+from .models   import Cart, Ingredient, User
+
+from utils import login_decorator
+
+class CartListView(View):
+    @login_decorator
+    def post(self,request):
+        try:
+            data = json.loads(request.body)
+            user = request.user
+
+            if data['count'] <= 0:
+                return JsonResponse({"message": "LESS_THAN_MINIMUM_QUANTITY"}, status=400)
+
+            duplicate_user = Cart.objects.filter(user_id=user.id)
```

Comment:
`.objects` manage class의 `filter()` method를 호출하면, 반환되는 객체는 
`QuerySet`(일종의 리스트라고 생각하시면 됩니다.)에 `Cart Instance`(테이블의 한 row 또는 record라고도 합니다.)가 담겨 있는 형태입니다.

그렇기 때문에 변수에 할당할 때, 보다 명확한 변수명을 사용해주셔야 합니다.

`<QuerySet : [ <Cart 1>, <Cart2> ...]`

```suggestion
            carts = Cart.objects.filter(user_id=user.id)
```

Reply:
수정하겠습니다!